Makefile

```diff
@@ -19,7 +19,15 @@ GOLANGCI_LINT_VERSION ?= "v1.37.0"
 REDIS_VERSION ?= "5.0.8"
 TOOLS_MOD_DIR := ./internal/tools
 
-BUILD_TAGS:=brotli
+BUILD_TAGS:=
+
+# the original Makefile had brotli alwas included
+# for this historical reason:
+#  - brotli is an *opt-out* feature (included by default, use NO_LIBBROTLI to *not* include it),
+#  - the other libs are opt-in features (not included by default, use USE_LIBSODIUM or USE_LZO to include them)
+ifndef NO_LIBBROTLI
```

Comment:
Let's move the optional brotli to the separate PR?

Reply:
done #1379